build.sbt

```diff
@@ -25,9 +24,15 @@ lazy val root = project
   .settings(noPublishSettings)
 
 lazy val core = myCrossProject("core")
+  .settings(
+    crossScalaVersions += "2.13.0"
```

Comment:
Haven't used the cross builder plugin before, is this how we add 2.13 as a compiler target?

Reply:
I think this is not needed. You only need to add 2.13.0 to `.travis.yml`.

Reply:
Thanks, removed.